File: test11.py
def compare_triplets(a, b):
    count, count_two = 0, 0;
    for i in range(3):
        if a[i] > b[i]:
            count += 1
        elif a[i] < b[i]:
            count_two += 1

    return [count, count_two]

a, b = [5, 6, 7], [3, 7, 10];
compare_triplets(a, b)

File: test11.rb
def compare_triplets(a, b)
    count = 0
    count_two = 0
    3.times do |i|
        if a[i] > b[i]
            count += 1
        elsif a[i] < b[i]
            count_two += 1
        end
    end
    p [count, count_two]
end

a = [3, 7, 10]
b = [5, 6, 7]
compare_triplets(a, b)